Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "frequenz-microgrid-component-graph-rs"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "component_graph"
```

Comment:
Is this name top-level or namespaced? If it is not namespaced, wouldn't it make sense to put it inside a `frequenz` namespace like we do in Python? Also mirroring the package name, maybe even `frequenz/microgrid` / `frequenz::microgrid`?

Reply:
We can't have different libraries in Rust under the same namespace (module).  I can change the library name to `frequenz_component_graph`, but then we'd have to:

```rust
use frequenz_component_graph;

let graph = frequenz_component_graph::ComponentGraph(...);
```

or 

```rust
use frequenz_component_graph as cg;
```

It depends on how strongly you feel about this.

Reply:
I see. IMHO `component_graph` without nothing else sounds a bit too generic. I'm not sure what is the common approach to this for companies that maybe have many open source rust libraries. From the code perspective, if it is easy enough to create a short alias as you showed, I guess we can live with the long name. Otherwise I noticed in some places we seem to use at least `fz_` as a prefix/namespace for our stuff, maybe `fz_component_graph` would be a OK-ish balance if having long library names is not popular at all in rust?

I just did a quick search in crates.io for google stuff, and even when it doesn't seem to be official, most libraries seem to use a full `google_cloud_xxx` or `googleapis_tonic_google_firestore_v1` as module, so I guess `frequenz_microgrid_component_graph` doesn't look that alien to rust. I would go with that to make it consistent with what we do in Python, but I could also live with `fz_component_graph` if it were necessary.

Reply:
done